server/main.lua

```diff
@@ -13,136 +14,107 @@ function ExploitBan(id, reason)
         'qb-management'
     })
 
-    TriggerEvent('qb-log:server:CreateLog', 'bans', "Player Banned", 'red', string.format("%s was banned by %s for %s", GetPlayerName(id), 'qb-management', reason), true)
+    TriggerEvent('qb-log:server:CreateLog', 'bans', 'Player Banned', 'red', string.format('%s was banned by %s for %s', GetPlayerName(id), 'qbx_management', reason), true)
 
-    DropPlayer(id, "You were permanently banned by the server for: Exploiting")
+    DropPlayer(id, 'You were permanently banned by the server for: Exploiting')
 end
 
+
 ---@param account string
 ---@return number
-function GetAccount(account)
+lib.callback.register('qbx_management:server:getAccount', function(_, account)
 	return exports['Renewed-Banking']:getAccountMoney(account) or 0
-end
-
-exports('GetAccount', GetAccount)
-exports('GetGangAccount', GetAccount)
+end)
 
 ---@param account string
 ---@param amount number
 function AddMoney(account, amount)
-	if not Accounts[account] then
-		Accounts[account] = 0
-	end
-
-	Accounts[account] += amount
-	exports['Renewed-Banking']:addAccountMoney(account, amount)
+	exports['Renewed-Banking']:addAccountMoney(account, amount, 'qbx_management')
 end
 
 ---@param account string
 ---@param amount number
 ---@return boolean
 function RemoveMoney(account, amount)
-
 	if amount <= 0 then return false end
-    if not Accounts[account] then
-        Accounts[account] = 0
-    end
-
-    local isRemoved = false
-
-    if Accounts[account] >= amount then
-        Accounts[account] -= amount
-        isRemoved = true
-    end
-
-	exports['Renewed-Banking']:removeAccountMoney(account, amount)
-	return isRemoved
+	return exports['Renewed-Banking']:removeAccountMoney(account, amount, 'qbx_management')
 end
 
-MySQL.ready(function ()
-	local funds = MySQL.query.await('SELECT job_name, amount FROM management_funds')
-	if not funds then return end
-
-	for i = 1, #funds do
-		local v = funds[i]
-		Accounts[v.job_name] = v.amount
-	end
-end)
-
 ---@param src number
 ---@param amount number
----@param pDataType 'job'|'gang'
----@param type 'boss'|'gang'
+---@param type 'job'|'gang'
 ---@param reason string
 ---@return boolean success
 ---@return Player? player populated if successful
 ---@return string? accountName populated if successful
-function WithdrawMoney(src, amount, pDataType, type, reason)
+function WithdrawMoney(src, amount, type, reason)
```

Comment:
```suggestion
local function WithdrawMoney(src, amount, type, reason)
```

Since we are going to 1 resource file we need all of these functions to be localized

Reply:
I forgot to add the suggestion here but if you make global to local functions make sure to camalCase them

Reply:
Changed.